src/devtools/views/SelectorEdit.html

```diff
@@ -32,6 +32,9 @@
 			</select>
 		</div>
 	</div>
+	<div class="form-group">
+		<input type="hidden" name="uuid" value="{{selector.uuid}}" disabled />
```

Comment:
I believe uuid should be displayed somewhere

Reply:
Update: we may use incrementing int counter as selector uid. Root is 0, N-th selector added gets uid N.